include/sempr/entity/SemanticEntity.hpp

```diff
@@ -0,0 +1,340 @@
+#ifndef SEMPR_ENTITY_SEMANTICENTITY_HPP
+#define SEMPR_ENTITY_SEMANTICENTITY_HPP
+
+#include <odb/core.hxx>
+#include <odb/wrapper-traits.hxx>
+#include <sempr/entity/RDFEntity.hpp>
+#include <memory>
+
+#include <Soprano/Soprano> // for the LiteralValue conversion functions
+#include <type_traits>
+#include <sempr/core/RDF.hpp>
+
+namespace sempr {
+
+namespace rdf {
+namespace traits {
```

Comment:
Maybe. Any suggestions?

Reply:
In principle: Yes. But this leads to changes in everything that uses sempr, again... Maybe postpone it until we do a complete overhaul of the structure? I'm not really happy with the  overall setup of directories and namespaces...

For now I'm fine with those traits being defined in the SemanticEntity.hpp, since only the SemanticEntity uses them.

Reply:
Ok for now its fine.